announcement/views/announcement_views.xml

```diff
@@ -115,20 +118,23 @@
                 </button>
             </xpath>
             <group name="announcement_validity" position="before">
-                <group name="announcement_scope">
+                <group
+                    name="announcement_scope"
+                    attrs="{'invisible': [('is_general_announcement', '=', True)]}"
+                >
                     <field name="announcement_type" />
                     <field
                         name="specific_user_ids"
                         string="Users"
                         options="{'no_create': True}"
-                        attrs="{'invisible': [('announcement_type', '!=', 'specific_users')], 'required': [('announcement_type', '=', 'specific_users')]}"
+                        attrs="{'invisible': [('announcement_type', '!=', 'specific_users')], 'required': [('announcement_type', '=', 'specific_users'), ('is_general_announcement', '=', False)]}"
```

Comment:
Is this new visibility rule needed at all when you're setting it up on the parent node?

Reply:
That rule is not for visibility, it is for required.

Reply:
Oh, yeah. In any case, It doesn't make sense to me. Can you explain it?

Reply:
Going back to analyse everything, in the logic I was considering at the beginning if it was necessary to take into account this condition to override that the field was required and let the operation for a general announcement. With the improvements and changes made it is not necessary. Thanks

Reply:
This attributes are for user interaction. If you're hiding the fields when you check that checkbox there's no possible user interaction. So no behaviour change is need at this level IMO